packages/rocketchat-ui/client/components/header/header.js

```diff
@@ -104,25 +104,38 @@ Template.header.helpers({
 });
 
 Template.header.events({
-	'click .iframe-toolbar button'() {
+	'click .iframe-toolbar .js-iframe-action'(e) {
 		fireGlobalEvent('click-toolbar-button', { id: this.id });
+		$('button', e.currentTarget).blur();
```

Comment:
`e.currentTarget.querySelector('body').blur()` should match the required browser supported.